airbyte-integrations/connectors/destination-firebolt/destination_firebolt/spec.json

```diff
@@ -0,0 +1,74 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/firebolt",
+  "supported_destination_sync_modes": [
+    "overwrite", "append"
+  ],
+  "supportsIncremental": true,
+  "supportsDBT": true,
+  "supportsNormalization": false,
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Firebolt Spec",
+    "type": "object",
+    "required": ["username", "password", "database"],
+    "additionalProperties": false,
+    "properties": {
+      "username": {
+        "type": "string",
+        "title": "Username",
+        "description": "Firebolt email address you use to login.",
+        "examples": ["username@email.com"],
+        "order": 0
+      },
+      "password": {
+        "type": "string",
+        "title": "Password",
+        "description": "Firebolt password.",
+        "airbyte_secret": true,
+        "order": 1
+      },
+      "account": {
+        "type": "string",
+        "title": "Account",
+        "description": "Firebolt account to login."
+      },
+      "host": {
+        "type": "string",
+        "title": "Host",
+        "description": "The host name of your Firebolt database.",
+        "examples": ["api.app.firebolt.io"]
+      },
+      "database": {
+        "type": "string",
+        "title": "Database",
+        "description": "The database to connect to."
+      },
+      "engine": {
+        "type": "string",
+        "title": "Engine",
+        "description": "Engine name or url to connect to."
+      },
+      "s3_bucket": {
+        "type": "string",
+        "title": "S3 bucket name",
+        "description": "The name of S3 bucket if using the External Table strategy."
+      },
+      "s3_region": {
+        "type": "string",
+        "title": "S3 region name",
+        "description": "Region name of the S3 bucket.",
+        "examples": ["us-east-1"]
+      },
+      "aws_key_id": {
+        "type": "string",
+        "title": "AWS Key ID",
+        "description": "AWS access key granting read and write access to S3."
+      },
+      "aws_key_secret": {
+        "type": "string",
+        "title": "AWS Key Secret",
+        "description": "Corresponding secret part of the AWS Key"
+      }
+    }
```

Comment:
I mean will it be a better UX if we use something similar to `loading_method` in bigquery connection create the specification. This will create a selection and you have to choose out of it.

Reply:
I was following the Redshift example, but looks like Bigquery is more fleshed out. Made the change.